main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+func calculateSum(number int) int {
+	sum := 0
+	for i := 1; i <= number; i++ {
+		sum += i
+	}
+	return sum
+}
+func hello(w http.ResponseWriter, req *http.Request) {
+	myURL := "https://localhost:8090/?n=10"
+	parsedURL, err := url.Parse(myURL)
+	if err != nil {
+		fmt.Println("Can't parse URL", err)
+		return
+	}
+	parts := strings.Split(parsedURL.RawQuery, "=")
```

Comment:
There should be a parsed query object. You shouldn't have to split here. https://pkg.go.dev/net/url#URL.Query